.gitignore

```diff
@@ -1 +1,2 @@
 /target
+/test-apps
```

Comment:
This shouldn't be in the .gitignore, since the test-apps should be included in the repository.